### PR TITLE
Additions of Skyrim (LE) and Fallout 3 

### PR DESCRIPTION
**Pull request by SirStig**
Additions:

- Added Skyrim (LE) post deploy and .desktop.

- Added Fallout 3 post deploy and .desktop.

Changes:

- Changed Skyrim SE post deploy and .sh names to specify it is Skyrim SE

Fixes:

- Found code in Falloutnv-post-deploy.sh that was pointing towards Fallout 4 and not Fallout NV. Corrected.

Notes:

Have not had anytime to download these three separate games and test everything out but in theory everything should work properly, Fallout 3 if it has any extra files as Fallout NV will not work then but never modded any of the Fallouts so I have no experience in this field.

### CODE REVIEW

**Comment by SirStig** [comment]
Think the Post Deploy files should be put into a Post Deploy folder on the desktop as to not crowd it.

**Comment by pikdum** [comment]
Parenthesis are a bit weird in file names, so I'd do this:
* keep the skyrim SE one the same, so it won't break existing shortcuts
* name the skyrim LE one skyrimle-post-deploy.sh and skyrimle-post-deploy.desktop

Should be clear enough.

**Comment by SirStig** [comment]
> Think the Post Deploy files should be put into a Post Deploy folder on the desktop as to not crowd it.

Also forgot to add the falloutnv-pre-deploy.desktop to the desktop

File: vortex/skyrim(SE)-post-deploy.desktop [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env xdg-open
 [Desktop Entry]
-Name=Skyrim Post-Deploy
-Exec=/home/deck/.pikdum/steam-deck-master/vortex/skyrim-post-deploy.sh
+Name=Skyrim SE Post-Deploy
```
Comment (pikdum) [inline vortex/skyrim(SE)-post-deploy.desktop:3]:
```suggestion
Name=Skyrim SE/AE Post-Deploy
```

Bit more clear.

**Comment by SirStig** [comment]
> Parenthesis are a bit weird in file names, so I'd do this:
> 
>     * keep the skyrim SE one the same, so it won't break existing shortcuts
> 
>     * name the skyrim LE one skyrimle-post-deploy.sh and skyrimle-post-deploy.desktop
> 
> 
> Should be clear enough.

Sure, will do!

**Comment by pikdum** [comment]
> Think the Post Deploy files should be put into a Post Deploy folder on the desktop as to not crowd it.

Desktop is fine for now, people can just delete shortcuts they aren't using.

**Comment by SirStig** [comment]
> > Think the Post Deploy files should be put into a Post Deploy folder on the desktop as to not crowd it.
> 
> Desktop is fine for now, people can just delete shortcuts they aren't using.

True, was thinking of changing everything to Symlinks, but not sure how much that would break anything.

File: vortex/.DS_Store
Comment (pikdum) [inline vortex/.DS_Store:1]: Looks like you accidentally committed this and another .DS_Store file.

Could add .DS_Store to a .gitignore.

Comment (SirStig) [reply vortex/.DS_Store:1]: Oops, yeah MacOS has unneeded files forgot to deselect them haha

**Comment by pikdum** [comment]
> True, was thinking of changing everything to Symlinks, but not sure how much that would break anything.

Yeah, I was playing around with symlinks a bit too for all of this.
Works well if you know the paths, but not sure how I'd handle migrating people over to it yet.

These post-deploy scripts seems easier for people to understand too, so I'll probably leave it alone for now.

**Comment by SirStig** [comment]
> > True, was thinking of changing everything to Symlinks, but not sure how much that would break anything.
> 
> Yeah, I was playing around with symlinks a bit too for all of this. Works well if you know the paths, but not sure how I'd handle migrating people over to it yet.
> 
> These post-deploy scripts seems easier for people to understand too, so I'll probably leave it alone for now.

Honestly, never really bothered me running the scripts. Takes two seconds. Symlinks would be easy in theory but would require finding and checking where the files are, if they are at all. Would have to run the script once anyway. 

Easiest thing to do would be to add a script that runs when you launch vortex, checks for games that are installed, creates symlinks for them if they don't exist.

**Review by pikdum — APPROVED** [review]

**Pull request merged**